Examples/jbang-catalog.json

```diff
@@ -0,0 +1,50 @@
+{
+  "catalogs": {
+    "example": {
+      "catalog-ref": "C:\\Users\\santo\\Git-projects\\core-edge-chains\\EdgeChains\\Examples\\jbang-catalog.json"
+    }
```

Comment:
I have added local path while creating catalog. Will change it to add the actual github url path for `jbang-catalog.json` once the PR is finalized